### ISSUE TITLE
ChromeOptions.setAcceptInsecureCerts(boolean) is missing

### ISSUE THREAD

**Issue by SiKing** [SeleniumHQ/selenium#4791]
OS:  ALL

Selenium Version:  3.6.0

Browser:  Chrome

Browser Version:  ANY

In Selenium 3.6.0 `ChromeDriver(Capabilities)` has been deprecated in favour of `ChromeDriver(ChromeOptions)`. However `ChromeOptions` class is missing `setAcceptInsecureCerts(boolean)` method. The only way to accomplish this functionality currently is to use the deprecated:
```
DesiredCapabilities capabilities = DesiredCapabilities.chrome();
capabilities.setAcceptInsecureCerts(true);
ChromeDriver(capabilities);
```

**Comment by SiKing** [comment]
I guess you can use `chromeOptions.setCapability(CapabilityType.ACCEPT_INSECURE_CERTS, true);`, but seems like the other Drivers have the setAcceptInsecureCerts method. Why not Chrome?

**Comment by p0deje** [comment]
What language do you use?

**Comment by SiKing** [comment]
Java

**Comment by lmtierney** [comment]
I only see FirefoxOptions having this method

**Comment by barancev** [comment]
Yes, we'll add this method, as well as some other similar ones.

**Comment by barancev** [comment]
Fixed by 2afcc4c81d863076b98af2b95f8cdffb1b865dda, to be released in version 3.7
